cmake/HPX_SetupBoost.cmake

```diff
@@ -29,12 +30,24 @@ set(Boost_ADDITIONAL_VERSIONS
     "1.57.0" "1.57")
 set(Boost_MINIMUM_VERSION "1.61" CACHE  INTERNAL "1.61" FORCE)
 
-set(__boost_libraries)
+set(__boost_libraries program_options system)
```

Comment:
The detection of Boost.ProgramOptions is now done in the program_options module

Reply:
right !